solana/tests/anchor.test.ts

```diff
@@ -285,21 +295,26 @@ describe("example-native-token-transfers", () => {
             extraAccountMetaList: extraAccountMetaListPDA,
             tokenProgram: TOKEN_PROGRAM,
             associatedTokenProgram: spl.ASSOCIATED_TOKEN_PROGRAM_ID,
-            systemProgram: SystemProgram.programId,
+            systemProgram: anchor.web3.SystemProgram.programId,
           })
           .instruction();
 
-      const transaction = new Transaction().add(
+      const transaction = new anchor.web3.Transaction().add(
         initializeExtraAccountMetaListInstruction
       );
       transaction.feePayer = payer.publicKey;
       const { blockhash } = await connection.getLatestBlockhash();
       transaction.recentBlockhash = blockhash;
 
       transaction.sign(payer);
-      await sendAndConfirmTransaction(connection, transaction, [payer], {
-        commitment: "confirmed",
-      });
+      await anchor.web3.sendAndConfirmTransaction(
+        connection,
+        transaction,
+        [payer],
+        {
+          commitment: "confirmed",
```

Comment:
I usually prefer an explicit commitment level for more transparency but if removing it makes it easier to follow, then I can remove it.

Reply:
It's not about making something easier to follow, but rather things can get mixed up. If you instantiate your connection to use a different commitment level then all hell can break loose. E.g. you query blockhashes that exist for a lower level but aren't yet known at a higher finality level. By always relying on the commitment level of the connection, you avoid this sort of mishap (and I've seen this happen in practice several times where you get rather vexing errors because the code looks like everything should be fine).

Reply:
I agree. That makes sense, thanks for clarifying!